example/example.map

```diff
@@ -158,7 +164,7 @@ MAP
 
     CONNECTIONTYPE OGR
     CONNECTION "polygons.geojson"
-    DATA "select fid as msfid, * from polygons"
+    DATA "select fid, name from polygons"
```

Comment:
Didn't we want a direct table reference here for performance (and add the fid manually to the gpkg)?

Reply:
we this is just a example with the example file (geojson)

Reply:
I understand, but since you are editing this anyways, it would be nice to keep this in line with our 'golden standard' (just to prevent confusion).

Reply:
or is `select fid, name from polygons` something from the mapserver master?

Reply:
in that case I agree